.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
Not sure how I feel about trailing commas as they aren't a think in our other codebases but that's neither here nor there.

Reply:
Ah good point, I'll remove that.